docs/release-notes/self-hosted/self-hosted-v12.0.0.md

```diff
@@ -0,0 +1,106 @@
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.1
+---
+
+# Self-hosted v12.0.0
+
+These release notes are for [Codacy Self-hosted v12.0.0](https://github.com/codacy/chart/releases/tag/12.0.0){: target="_blank"}, released on June 21, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+Jira issues without release notes
+-   https://codacy.atlassian.net/browse/TS-110 TODO review
+
+Jira issues with disabled release notes
+-   https://codacy.atlassian.net/browse/TS-369 TODO Notify Paulo that we're ignoring this one: https://codacy.atlassian.net/browse/TS-369?focusedCommentId=56541
+-   https://codacy.atlassian.net/browse/TS-356 TODO we may want to notify this: ESlint: install plugin vue-scoped-css
+-->
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v12.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v12.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:<!--TODO Update CLI tool versions-->
+
+    -   [Codacy Analysis CLI MAJOR.MINOR.PATCH](https://github.com/codacy/codacy-analysis-cli/releases/tag/MAJOR.MINOR.PATCH)
+    -   [Codacy Coverage Reporter MAJOR.MINOR.PATCH](https://github.com/codacy/codacy-coverage-reporter/releases/tag/MAJOR.MINOR.PATCH)
+
+## Breaking changes
+
+If you're updating Kubernetes to v1.25, upgrade to Codacy Self-hosted v12.0.0 before doing so, as some Kubernetes endpoints have been deprecated and may lead to service outage if used with earlier versions of Codacy Self-hosted.
+
+If using the monitoring functionality [Grafana + Prometheus + Loki](https://docs.codacy.com/chart/configuration/monitoring/#setting-up-monitoring-using-grafana-prometheus-and-loki), ensure Pod Security Policies are disabled for Loki and Promtail by following the relevant update instructions ([Loki](https://docs.codacy.com/chart/configuration/monitoring/#2-installing-loki), [Promtail](https://docs.codacy.com/chart/configuration/monitoring/#3-installing-promtail)) after upgrading to Codacy Self-hosted v12.0.0 and before updating Kubernetes to v1.25.
+
+If you have set `scheduler.enable = "true"` in `values.yaml`, before migrating to Codacy Self-hosted v12.0.0:
+
+-   Remove scheduler settings from `values.yaml`
+-   In `worker-manager`, ConfigMap set CONFIG_FORCE_codacy_kubernetes_scheduler_name: default-scheduler 
+-   Restart `worker-manager` deployment
+-   Remove any pending worker Pods
```

Comment:
@tfrgoncalves A couple notes:

`scheduler.enable = "true"`
I'm checking [`values.yaml`](https://github.com/codacy/chart/blob/master/codacy/values.yaml) and see lots of `enabled` keys but no `enable`. Just double-checking to confirm there's no typo here.

`ConfigMap set CONFIG_FORCE_codacy_kubernetes_scheduler_name: default-scheduler`
Can you provide more context on how this is run and provide an example? Just so I know how to format these commands (is it from a command line, config file, ...)

Reply:
Rechecking always makes sense. We have [this](https://github.com/codacy/worker-manager-sh/blob/3515fe90883b7c21a1b34c6fa98c8fe93af9aea7/conf/reference.conf#L449) on the documentation and [here](https://github.com/codacy/worker-manager-sh/pull/9/files#diff-5f9d82490629d7c4b8aab497e5a44971aee6703afccf21ca665498a8cd8c5acaL55) Lorenzo is removing it.


Regarding the ConfigMap, I am proposing a 'manual' edition of the Configmap. We could provide a kubectl command as I mentioned before, but that may pose some dangers. I will leave all existing options below.

Assuming codacy is installed in namespace 'codacy' as usual:
Manual approach: `kubectl edit configmap worker-manager --namespace codacy`, change said value to 'default-scheduler' and save.

Automatic approach: `kubectl patch configmap/worker-manager --namespace codacy --type merge --patch '{"data":{"CONFIG_FORCE_codacy_kubernetes_scheduler_name":"default-scheduler"}}'`

Shorter automatic approach: `kubectl patch configmap/worker-manager -n codacy --type merge -p '{"data":{"CONFIG_FORCE_codacy_kubernetes_scheduler_name":"default-scheduler"}}'`

We may leave the last option with a note to contact support in case of need.